### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by isaka-lumato**

### CODE REVIEW

**Comment by isaka-lumato** [comment]
Hello @isaka-lumato 
This is a brilliant effort. You've really done an amazing job so far. In this PR, my major highlights include

the code works well
However, you need to make some required changes. Please check my comments below and fix them as appropriate.

Required Changes recycle

- Properly set up your linters
- have a proper PR description
- I feel like your readMe file could be more descriptive of your project, you can refer to your given template

Optional Changes 
N/A

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   <body>
     <h1 id="title">MICROVERSE SURVEY</h1>
-    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <p id="description">Thank you all for taking the time to help us improve the platform</p>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
     <script  src="js/index.js"></script>
```
Comment (teekaytech) [inline index.html:14]: _**Example of an inline comment:**_

You've added a `script` tag in the middle of the body tag. Although you might have your reasons, I think this is not the best place to put this tag. By convention, <script> tags are usually inserted either inside the `head` tag or just before closing the body tag. Kindly fix this :blush:

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @isaka-lumato :wave:

_This is a brilliant effort :clap: You've done really well reviewing your project. Although there's room for improvement, I don't want you to see this as criticizing your effort. I want you to see this feedback as a step in the right direction to help you understand better how to review code._

### **Required Changes** :recycle: 

- [ ] You've played well the role of a code reviewer. However, you need to keep the following in mind: 
  - It would have been better if you use the `Review Changes` feature in GitHub. You can find this button by clicking on the `Files changed` option on your PR page. This will give you access to other amazing GitHub features necessary to perform an interesting and educative code review. Check the gif image below for more illustration:
NEW
  - This project requires that you add as many inline comments as possible. In the worst-case scenario, you need to add a minimum of 2 comments. Kindly use GitHub features comments as specified [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md), paying special attention to point number 3 (inline comments).
  - I believe you should be able to find at least 2 things that can be improved on in your code. For instance, you can point out that "the PR summary is empty", "lack of <a href="https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md" rel="nofollow noreferrer noopener" target="_blank">semantic HTML tags use</a>", etc. These and others can be pointed out in your inline comments; you will definitely find something that could have been done better!

Check an example of an inline comment under the review.

### **Optional Changes** :zap: 
N/A


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

**Review by isaka-lumato — COMMENTED** [review]
Hello @isaka-lumato
This is a brilliant effort. You've really done an amazing job so far. In this PR, my major highlights include

the code works well
However, you need to make some required changes. Please check my comments below and fix them as appropriate.

Required Changes recycle

1. properly setup your linters
2. make good use of semantic tags like <header> <section> among others
3. kindly make changes to the issues addressed above

Optional Changes
N/A

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @isaka-lumato!

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

ok ok

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**